### ISSUE TITLE
Text file: differences between OpenWithEditor and OpenWith

### ISSUE THREAD

**Issue by alphaniner** [fman-community/fman#671]
I use gvim for Windows. Before I realized a text editor could be configured, I configured an 'open with' entry to open text files with gvim.

When I use the 'open with' method, gvim's working directory is the directory containing the file. When I use the 'edit' method, gvim's working directory is 'C:\Users\alphaniner\AppData\Local\fman\Versions\1.6.4`.

The issue is different formats of the file path. When using the 'open with' method the path gets transformed to human readable format (line 1820 of `core/commands/__init__.py`). When using the 'edit' method it doesn't.

I changed line 429 of `core/commands/__init__.py` from
```
popen_kwargs = strformat_dict_values(editor, {'file': path})
```
to
```
popen_kwargs = strformat_dict_values(editor, {'file': as_human_readable(url)})
```
to fix.

**Comment by mherrmann** [comment]
Thank you for the excellent research. The fix you suggested will go live with fman 1.6.5.

**Comment by mherrmann** [comment]
Fixed in [fman 1.6.5](https://fman.io/changelog#v1.6.5).
